components/MintPageContainer.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <Page
+    :class="[
+      'flex',
+      'flex-col',
+      'relative',
+      'items-center',
+      'justify-center',
+      'px-[20px]',
+      'pt-[38px]',
+      'lg:p-[16px]',
+    ]"
+  >
+    <!-- Email block -->
+    <div
+      v-if="!hasProvidedEmail"
+      :class="[
+        'flex',
+        'items-start',
+        'justify-start',
+        'gap-[8px]',
+        'px-[12px]',
+        'py-[8px]',
+        'bg-[#f1fbff]',
+        'border-[1px]',
+        'border-[#caedff]',
+        'rounded-[12px]',
+        'mb-[24px]',
+        'w-full',
+        'max-w-[600px]',
+      ]"
+    >
+      <IconAttention class="flex-shrink-0 text-[#a9d3e2]" />
+      <div class="flex flex-col">
+        <Label preset="p6" class="whitespace-pre-line">
+          <i18n path="NFTPortal.label.input.email" tag="div">
+            <template #link>
+              <a
+                v-t="$t('NFTPortal.label.input.link')"
+                :class="['underline', 'text-like-green']"
+                href="https://forms.gle/dwhq6Dny3DnDUEzg9"
+                target="_blank"
+              />
+            </template>
+          </i18n>
+        </Label>
+      </div>
+    </div>
+    <slot />
+    <div class="flex flex-col max-w-[600px]">
+      <div class="mt-[-20px]">
+        <AttentionsOpenLikerLandApp v-if="isUsingLikerLandApp" />
```

Comment:
We may want to refactor this to `isUsingWalletConnect` and support all wallet connect method in the future, since this component is used to remind people to open their app

Reply:
Also `isStateTransaction` is removed, this prompt is actually less useful UX wise
Old behaviour is this component only appears when signing is needed, and is hidden otherwise. so people are reminded when to wait and when to open the app to continue
Can we implement this behaviour here somehow?